libs/portals/shared-modules/delegations/src/components/access/AccessDeleteModal/AccessDeleteModal.tsx

```diff
@@ -41,7 +42,7 @@ export const AccessDeleteModal = ({
       getAuthScopeTree({
         variables: {
           input: {
-            domain: delegation.domain.name,
+            domain: delegation.domain?.name ?? null,
```

Comment:
Can´t we just skip making the `getAuthScopeTree` request if domain.name is null? Then we are not displaying any scopes.